test/unit/samples/model/qti/item/testToQTIWithDirAttributeInItemBody_expectedItemQti.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?><assessmentItem
+        xsi:schemaLocation=""
+     identifier="Item_1" title="" label="" adaptive="false" timeDependent="false" toolName="TAO">
+
+    
+    
```

Comment:
Do we need these empty lines?

Reply:
Method whitch we are testing toQti() returns xml is such format with empty lines and so if we want to compare it we need those lines in this saimple file.